unittests/fixtures/factory/work-dir/update_proxy.py

```diff
@@ -6,16 +6,16 @@
 """This file is specialized in updating a proxy file
 
 
- All information is passed via the environment;
-  so it has no arguments
+All information is passed via the environment;
+ so it has no arguments
 
- Env variables used:
-  HEXDATA - b2a_hex(proxy_data)
-  FNAME   - file name to update
+Env variables used:
+ HEXDATA - b2a_hex(proxy_data)
+ FNAME   - file name to update
 
- The python-related environment variables must also
-  be properly set
-  PATH, LD_LIBRARY_PATH, PYTHON_PATH
+The python-related environment variables must also
+ be properly set
+ PATH, LD_LIBRARY_PATH, PYTHON_PATH
```

Comment:
Any reason why `PATH, LD_LIBRARY_PATH, PYTHON_PATH` are on a separate line rather than continuing in the previous line (after `be previously set`). Is it again the max characters in a line limit?